apps/crn-server/src/data-providers/research-tags.data-provider.ts

```diff
@@ -104,8 +106,33 @@ export class ResearchTagSquidexDataProvider implements ResearchTagDataProvider {
       }),
     };
   }
+
+  async fetchAll(options: FetchAllOptions): Promise<ListResearchTagDataObject> {
```

Comment:
I am not sure if I would make this a new method, especially since the DataProvider interface clearly shows which 4 methods should be available. Maybe just make `all` a parameter for fetch?

Reply:
I thought the point of the interface was so our data providers implement core CRUD methods, but could also have other methods. If we don't want to have `fetchAll` in the provider then i think we could

- move it to the controller level
- move the fetching to the FE

If we were to use `all` would we pass this from the FE?

Reply:
controller is my preference, yes